### DISCUSSION TITLE
Add a concrete text to the invoice footer

### DISCUSSION THREAD

**Discussion by KKBastian** [Blair2004/NexoPOS#421] [Gastro 4.x Ideas & Issues] [answered]
Hi,

We want to ad a text to the invoice footer and it should not be editable in the system. I meant that it should be a text line which can not edit by any users of the system. Can we do it from the back end codes. How can we do that ?

**Comment by Blair2004** [comment, marked as answer]
That means you want to hardcode a value on the system. Unfortunately, we cannot do such a thing on the core. If you want to perform that modification on your own project, you should look here : https://github.com/Blair2004/NexoPOS-4x/blob/master/resources/ts/pages/dashboard/orders/ns-order-invoice.vue

**Comment by KKBastian** [comment]
Is this the receipt code? 
We tried to do some changes, but it didn't change the receipt.
Can I know the place to edit(code line)?

**Reply by Blair2004** [reply]
After editing the receipt you need to compile the code, the instructions are on the package.json. Note that this modification should be made by a developer who knows how to use Vue.js applications.
